### ISSUE TITLE
check 'website' manifest key is a valid URI

### ISSUE THREAD

**Issue by sbidoul** [OCA/pylint-odoo#121]
pypi checks the url key is a valid uri according to rfc3986, therefore addons with an invalid website key in the manifest cannot be published to pypi.

Hence, I'm suggesting the addition of the following check to pylint-odoo, similar to [what pypi does](https://github.com/pypa/pypi-legacy/blob/5937ede003477561bc48a041384eeb22091d30d7/webui.py#L2699-L2709):

```python
import rfc3986

website = manifest.get('website')
uri = rfc3986.uri_reference(website)
if not uri.is_valid(require_scheme=True, require_authority=True):
    # error
if uri.scheme not in {"http", "https"}:
    # error
```

**Comment by dreispt** [comment]
:+1:

**Comment by moylop260** [comment]
@jesuszapata could you help us to add this new check, please?

**Comment by JesusZapata** [comment]
@sbidoul 
Sorry for my long time answering

I try used `rfc3986` but this library not made the correct validation

![image](https://user-images.githubusercontent.com/1387970/27336820-b3ef0c5a-559e-11e7-81cb-6baf7312b780.png)

I proposal used the `validators` http://validators.readthedocs.io/ library 

![image](https://user-images.githubusercontent.com/1387970/27336906-f2c8018e-559e-11e7-92c3-0911a4eba7c1.png)

Are you agree ?

**Comment by pedrobaeza** [comment]
Maybe first library "failed" because they are 2 valid URLs. Can you detect a comma and pass the validator?

**Comment by sbidoul** [comment]
I think I saw the rf3986 used in pypi source code. We need to use the same
check as pypi. I'll check later.

**Comment by moylop260** [comment]
@sbidoul 
Is possible use a built-in python library like as `urllib` to detect a valid url?
In order to reduce the packages to install

**Comment by sbidoul** [comment]
The code sample in the issue description is the exact same used on pypi. So for me it's good enough to use that one. Note it does raise an error if there is a space after the comma. 

But I of course a stricter validation is always nice.
